Assets/__Scripts/Platforms/Track Rings/ITrackLaneRingsManager.cs

```diff
@@ -0,0 +1,10 @@
+﻿using UnityEngine;
+
+public abstract class ITrackLaneRingsManager : MonoBehaviour
```

Comment:
Again, the `I` prefix is for defining interfaces.

For this and `IRotatingLights`, it is perfectly possible to convert to an interface. You would need to make sure that the inheriting classes re-inherit `MonoBehaviour`.

Reply:
Renamed class